flow-typed/npm/downshift_vx.x.x.js.flow

```diff
@@ -0,0 +1,180 @@
+/**
+ * Flowtype definitions for index
+ * Generated by Flowgen from a Typescript Definition
```

Comment:
Should we have a script in the package.json for regenerating this file?

Potentially it'd be great if we do this as a precommit hook so we don't have to run that script manually:

```js
// lint-staged.config.js
const baseConfig = require('kcd-scripts/config').lintStaged
module.exports = Object.assign(baseConfig, {
  linters: Object.assign(baseConfig.linters, {
    '**/typings/**/*.d.ts': [
      'npm run generate-flow',
      'git add flow-typed'
    ]
  })
})
```

What do you think?

Reply:
The problem is that this generator doesn't work. I needed to edit generated file manually. I will fix all other your notes